skyvern/core/script_generations/skyvern_page.py

```
@@ -683,6 +684,120 @@ async def extract(
+
+            # Fallback mode - try selector first, use AI if it fails
+            download_button = page.locator("#download-btn", prompt='find "download invoices" button')
+            await download_button.click()
+
+            # With Playwright parameters
+            submit_button = page.locator(prompt="find submit button", has_text="Submit")
+            await submit_button.click()
+            ```
+        """
+        if not selector and not prompt:
+            raise ValueError("Missing input: pass a selector and/or a prompt.")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)